src/test/enhetstester/kotlin/no/nav/familie/ks/sak/statistikk/stønadsstatistikk/StønadsstatistikkServiceTest.kt

```diff
@@ -177,6 +182,100 @@ internal class StønadsstatistikkServiceTest {
+                emptyList(),
+            )
+
+        val andelTilkjentYtelseBarn1Periode2 =
+            AndelTilkjentYtelseMedEndreteUtbetalinger(
+                lagAndelTilkjentYtelse(
+                    behandling = behandling,
+                    stønadFom = OVERGANGSORDNING_UTBETALINGSMÅNED.førsteDagIInneværendeMåned().toYearMonth().plusMonths(1),
+                    stønadTom = OVERGANGSORDNING_UTBETALINGSMÅNED.sisteDagIInneværendeMåned().toYearMonth().plusMonths(1),
+                    sats = 3750,
+                    aktør = barn1.aktør,
+                    periodeOffset = 1,
+                    ytelseType = YtelseType.OVERGANGSORDNING,
+                ),
+                emptyList(),
+            )
+
+        val andelTilkjentYtelseBarn2 =
+            AndelTilkjentYtelseMedEndreteUtbetalinger(
+                lagAndelTilkjentYtelse(
+                    behandling = behandling,
+                    stønadFom = OVERGANGSORDNING_UTBETALINGSMÅNED.førsteDagIInneværendeMåned().toYearMonth().minusMonths(4),
+                    stønadTom = OVERGANGSORDNING_UTBETALINGSMÅNED.sisteDagIInneværendeMåned().toYearMonth().plusMonths(2),
+                    sats = 1054,
+                    aktør = barn2.aktør,
+                ),
+                emptyList(),
+            )
+
+        val andelTilkjentYtelseSøker =
+            AndelTilkjentYtelseMedEndreteUtbetalinger(
+                lagAndelTilkjentYtelse(
+                    behandling = behandling,
+                    stønadFom = barn2.fødselsdato.førsteDagINesteMåned().toYearMonth(),
+                    stønadTom = barn2.fødselsdato.plusYears(3).toYearMonth(),
+                    sats = 50,
+                    aktør = barn2.aktør,
+                ),
+                emptyList(),
+            )
+
+        val andelerTilkjentYtelse =
+            listOf(
+                andelTilkjentYtelseBarn1,
+                andelTilkjentYtelseBarn1Periode2,
+                andelTilkjentYtelseBarn2,
+                andelTilkjentYtelseSøker,
+            )
+
+        every { behandlingHentOgPersisterService.hentBehandling(any()) } returns behandling
+        every { persongrunnlagService.hentAktivPersonopplysningGrunnlagThrows(any()) } returns personopplysningGrunnlag
+        every { vedtakService.hentAktivVedtakForBehandling(any()) } returns vedtak
+        every { personopplysningerService.hentLandkodeUtenlandskBostedsadresse(any()) } returns "DK"
+        every { andelerTilkjentYtelseOgEndreteUtbetalingerService.finnAndelerTilkjentYtelseMedEndreteUtbetalinger(any()) } returns
+            andelerTilkjentYtelse
+        every { vilkårsvurderingService.hentAktivVilkårsvurderingForBehandling(any()) } returns vilkårsVurdering
+        every { persongrunnlagService.hentBarna(any()) } returns personopplysningGrunnlag.barna
+        every { kompetanseService.hentKompetanser(any()) } returns emptyList()
+
+        val vedtakDvh = stønadsstatistikkService.hentVedtakDVH(1L)
```

Comment:
Vuder arrange-act-assert kommentarer for å gjøre det lettere å se at det er denne linjen som blir testet